src/docfx/data/ContextObject.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
These schemas shall go to Microsoft.Docs.Template.csproj

Reply:
Yes, will integration with Template later, TODO already added